src/pages/settings/AboutPage/AboutPage.js

```diff
@@ -95,64 +96,56 @@ function AboutPage(props) {
         }));
     }, [translate, waitForNavigate]);
 
+    const overlayContent = () => (
+        <View style={[styles.pAbsolute, styles.w100, styles.h100, styles.justifyContentEnd, styles.pb5]}>
+            <Text
+                selectable
+                style={[styles.textLabel, styles.textIvoryLight, styles.alignSelfCenter]}
```

Comment:
This is defining an anonymous component. On every render we define a new component and this may cause performance degradation as react will remove the child and create a completely new one. This is like changing the key prop every render. Let's use `useCallback` here so we keep the same component.